plugins/balancedLevelProvider.go

```diff
@@ -93,6 +108,26 @@ func validateSpread(spread float64) {
 
 // GetLevels impl.
 func (p *balancedLevelProvider) GetLevels(maxAssetBase float64, maxAssetQuote float64) ([]api.Level, error) {
+	log.Printf("Last base balance was: %v", p.lastMaxAssetBase)
```

Comment:
- we typically start these log lines with a lowercase character
- use `%.7f` instead of `%v`
- add `\n` after `%.7f` -- we always add newlines to printf statements as that will help immensely when we add a new logging framework
- we should also print the current balances for easy viewing

putting this all together, it should be: 
```
log.Printf("balance for base asset: last=%.7f, current=%.7f\n", p.lastMaxAssetBase, maxAssetBase)
```